api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -796,6 +796,215 @@
+	</changeSet>
+	<changeSet author="vshankar" id="201610131530-1.2">
+		<preConditions onFail="MARK_RAN">
+			<columnExists tableName="cohort_member" columnName="voided"/>
+		</preConditions>
+		<addNotNullConstraint tableName="cohort_member" columnName="voided" columnDataType="boolean"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201610131530-2">
+		<preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="voided_by"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="voided_by" type="int"/>
+		</addColumn>
+	</changeSet>
+	<changeSet author="vshankar" id="201610131530-3">
+		<preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="date_voided"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="date_voided" type="DATETIME"/>
+		</addColumn>
+	</changeSet>
+	<changeSet author="vshankar" id="201610131530-4">
+		<preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="void_reason"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="void_reason" type="varchar(255)"/>
+		</addColumn>
+	</changeSet>
+	<changeSet id="201610042145-2" author="vshankar">
+		<preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="uuid"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="uuid" type="char(38)"/>
+		</addColumn>
+	</changeSet>
+	<changeSet id="201610042145-2.1" author="vshankar">
+		<preConditions onFail="MARK_RAN"><or><dbms type="mysql"/><dbms type="oracle" /></or></preConditions>
+		<comment>Generating UUIDs for all rows in cohort_member table via built in uuid function.</comment>
+		<update tableName="cohort_member" >
+			<column name="uuid" valueNumeric="name-of-uuid-function" />
+			<where>uuid is null</where>
+		</update>
+		<modifySql dbms="mysql"><replace replace="name-of-uuid-function" with="UUID()"/></modifySql>
+		<modifySql dbms="oracle"><replace replace="name-of-uuid-function" with="SYS_GUID()"/></modifySql>
+	</changeSet>
+	<changeSet id="201610042145-2.2" author="vshankar">
+		<addNotNullConstraint tableName="cohort_member" columnName="uuid" columnDataType="char(38)"/>
+	</changeSet>
+    <changeSet id="201610242135-1" author="vshankar">
+		<addForeignKeyConstraint baseColumnNames="cohort_id" baseTableName="cohort_member" constraintName="parent_cohort" deferrable="false" referencedColumnNames="cohort_id" referencedTableName="cohort"/>
+	</changeSet>
+	<changeSet id="201610242135-2" author="vshankar">
+		<addForeignKeyConstraint baseColumnNames="patient_id" baseTableName="cohort_member" constraintName="member_patient" deferrable="false" initiallyDeferred="false" referencedColumnNames="patient_id" referencedTableName="patient"/>
+	</changeSet>
 	<changeSet id="20090402-1515-38-concept" author="bwolfe" dbms="mysql">
```

Comment:
patient_id and cohort_id fields previously had a default value of 0 which in my opinion is wrong, can you drop it please? Liquibase has `dropDefaultValue` tag